torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -1,44 +1,100 @@
 defmodule Torchx.NxDoctestTest do
+  @moduledoc """
+
+  # Why some doctests are excluded?
+
+  ## Not callbacks
+
+  * default_backend/2
+
+  ## Python rounds differently, than Elixir (for now)
+
+  * round/1
+
+  ## Result mismatch in less significant digits (precision issue)
+
+  * logistic/1
+
+  ## Callback or test requires explicity backend transfer to BinaryBackend
+
+  * to_binary/2
+  * to_flat_list/2
+  * random_uniform/4
+  * random_normal/4
+  * atan2/2
+
+  ## Not all options are supported
+
+  * argmax/2
+  * argmin/2
+
+  ## Unsigned ints not supported by Torch
+
+  * quotient/2
+  * mean/2
```

Comment:
For those we can move the tests that require unsigned integers to `test/nx_test_exs` so we can use them.